src/Api/Controllers/AccountsController.cs

```diff
@@ -799,16 +799,16 @@ public async Task<ApiKeyResponseModel> RotateApiKey([FromBody]ApiKeyRequestModel
             }
         }
         
-        [HttpPost("update-temp-password")]
-        public async Task PostUpdateTempPasswordAsync([FromBody]UpdateTempPasswordRequestModel model)
+        [HttpPut("update-temp-password")]
+        public async Task PutUpdateTempPasswordAsync([FromBody]UpdateTempPasswordRequestModel model)
```

Comment:
Is this going to cause regression issues in older clients?

Reply:
@cscharf I don't believe this made it out to any clients (most recent release 27 days ago). I added the `UpdateTempPasswordAsync` method end on July 22nd (14 days ago).